src/main/proto/confidence/flags/resolver/v1/types.proto

```diff
@@ -60,4 +60,20 @@ enum SdkId {
   SDK_ID_PYTHON_PROVIDER = 6;
   // Confidence OpenFeature GO Provider.
   SDK_ID_GO_PROVIDER = 7;
+  // Confidence OpenFeature Ruby Provider.
```

Comment:
curious why are we adding these types back?

Reply:
no all good 👍

Reply:
I tend to agree with you both that the pros of keeping the protos perfectly in sync are not as relevant as the cons of adding unused generated code to the client, as well as giving the client definitions that it is perhaps not meant to be using... I'd keep this small PR as it is, but it is an interesting topic though